stdlib/public/core/CMakeLists.txt

```diff
@@ -353,6 +353,9 @@ set(swiftCore_common_options
                   SWIFT_COMPILE_FLAGS
                     ${swift_stdlib_compile_flags} -Xcc -DswiftCore_EXPORTS ${SWIFT_STANDARD_LIBRARY_SWIFT_FLAGS})
 
+# Enable OSSA Modules for swift stdlib
```

Comment:
I think you can remove this since it comes for free with the changes in `stdlib/cmake/modules/AddSwiftStdlib.cmake`.

Reply:
Done